### PR TITLE
DSNS-179: updated the airflow for nci_s2_ard such that its logging will be to a path that reflects the date of processing instead of the date the task was scheduled. This gives debugging/ fault troubleshooting to be performed easier and accurately.

### PR DESCRIPTION
**Pull request by gordonyeong**

### CODE REVIEW

**Review by dunkgray — COMMENTED** [review]
Lets get the time in there and use the current format with something like
export CURRENT_DATE=$(date +'%Y%m%dT%H%M%S')

Plus have was is in dev for this file and what changes are being PR'ed into prod are identical

File: dags/nci_ard/nci_s2_ard.py [hunk 1]
```diff
@@ -64,7 +64,7 @@
 default_args = {
     "owner": "Duncan Gray",
     "depends_on_past": False,  # Very important, will cause a single failure to propagate forever
-    "start_date": datetime(2020, 8, 31),
+    "start_date": datetime(2020,6,2),
```
Comment (dunkgray) [inline dags/nci_ard/nci_s2_ard.py:67]: This doesn't look intentional

**Review by dunkgray — COMMENTED** [review]
This variable doesn't seem to be defined.
${logging_directory}

This is likely causing the NCI job e and o files to be written to lpgs home when currently executing from the dev branch.

-rw-r--r--   1 lpgs v10        331 May 26 14:27 ard_scene_select.e85172632
-rw-r--r--   1 lpgs v10        331 May 26 14:27 ard_scene_select.e85172607
-rw-r--r--   1 lpgs v10        331 May 26 14:40 ard_scene_select.e85173953
-rw-r--r--   1 lpgs v10  451214558 May 26 15:16 ard_scene_select.e85175302
-rw-r--r--   1 lpgs v10  451214568 May 26 17:01 ard_scene_select.e85188903
-rw-r--r--   1 lpgs u46          0 May 28 10:21 tmp_scene_paths_2023-05-28.csv
-rw-r--r--   1 lpgs u46          0 May 28 10:40 .wget-hsts
-rw-r--r--   1 lpgs u46          0 May 29 09:48 tmp_scene_paths_2023-05-27.csv
-rw-r--r--   1 lpgs u46          0 May 29 10:21 tmp_scene_paths_2023-05-29.csv
drwxrwxr-x   2 lpgs u46       8192 May 30 10:45 .logrotate
-rw-r--r--   1 lpgs u46          0 May 30 11:22 tmp_scene_paths_2023-05-30.csv
-rw-r--r--   1 lpgs v10          0 May 30 11:38 -e
drwxr-x---  50 lpgs v10      65536 May 30 12:02 .
[lpgs@gadi-login-03 ~]$ ls -latr

File: dags/nci_ard/nci_s2_ard.py (hunk 1 above)
Comment (gordonyeong) [reply dags/nci_ard/nci_s2_ard.py:67]: it was very late in the year thus i thought i revise it because we are getting closer to deployment.

Comment (dunkgray) [reply dags/nci_ard/nci_s2_ard.py]: Let's not do this. We don't really use the start date. It isn't part of this log change.

**Review by dunkgray — COMMENTED** [review]
Lets ditch - general_logging_dir
And have -o & -e be {{ params.base_dir }}${log_ext}

**Comment by gordonyeong** [comment]
hello,    @dunkgray  :) 

checked test run on dev and its logs. Works as expected.

https://airflow.dev.dea.ga.gov.au/log?dag_id=nci_s2_ard&task_id=wait_for_completion&execution_date=2023-05-30T07%3A39%3A01.956580%2B00%3A00&map_index=-1

https://github.com/GeoscienceAustralia/dea-airflow/pull/1265  is ready for review :)



thank you :)

**Review by dunkgray — APPROVED** [review]
looks good, thanks.

**Pull request merged**